package.json

```diff
@@ -17,5 +17,8 @@
     },
     "lint-staged": {
         "**/*": "prettier --write --ignore-unknown"
+    },
+    "dependencies": {
```

Comment:
I'm glad this is working now, but you should install this in a different spot. The top-level one is only for project-wide installation, which is basically our pre-commit hooks. `pretty` is  used only as a dev dependency in the `memory-viz` package (note that each package has its own `package.json` file).

You can do something like `npm i --save-dev --workspace=memory-viz <package>` to install and save a dev dependency in one of the individual packages.

Please do that, and also remove the dependency from the top-level `package.json`.